### PR TITLE
Fix CoerceUtil Bug

### PR DESCRIPTION
**Pull request by xiaoyao1991**
#260

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: elide-core/src/main/java/com/yahoo/elide/utils/coerce/CoerceUtil.java [hunk 1]
```diff
@@ -55,6 +54,11 @@ public static Object coerce(Object value, Class<?> cls) {
      */
     private static void setup() {
         BeanUtilsBean.setInstance(new BeanUtilsBean(new ConvertUtilsBean() {
+            {
+                // https://github.com/yahoo/elide/issues/260
+                // disable returning defaultValues when conversions fail
```
Comment (clayreimann) [inline elide-core/src/main/java/com/yahoo/elide/utils/coerce/CoerceUtil.java]: This note should be changed from a negation to an assertion like: `enable throwing when conversion fails`

**Comment by DeathByTape** [comment]
👍  after addressing @clayreimann  comment

**Comment by clayreimann** [comment]
👍 here

**Pull request merged**